src/main/java/org/jabref/gui/copyfiles/ExportLinkedFilesAction.java

```diff
@@ -76,7 +71,11 @@ private void startServiceAndshowProgessDialog(Task<List<CopyFilesResultItemViewM
     }
 
     private void showDialog(List<CopyFilesResultItemViewModel> data) {
-        Dialog<ButtonType> dlg = new Dialog<>();
+
+        CopyFilesDialogView dlg = new CopyFilesDialogView(databaseContext, new CopyFilesResultListDependency(data));
```

Comment:
@tobiasdiez  Here I am passing the instance of the object, but it doesn't display any data.